### DISCUSSION TITLE
Track Cpu and Memory Usage for each honeypot.

### DISCUSSION THREAD

**Discussion by deanXsaber** [telekom-security/tpotce#760] [Q&A]
Hi,

Is there a way to track the CPU and memory usage for each honeypot? 
My VM only has 8G memory, and the memory usage is very high all the time. I'm wondering whether I should turn off certain honeypots or not.

Btw, Thanks a lot for this wonderful project!

**Comment by t3chn0m4g3** [comment]
You can use `glances` or `cockpit` for that.

Thanks :bowtie:
